front/src/WebRtc/MediaManager.ts

```diff
@@ -318,6 +318,8 @@ export class MediaManager {
         const localScreenCapture = this.localScreenCapture;
         this.getCamera().then((stream) => {
             this.triggerStoppedScreenSharingCallbacks(localScreenCapture);
+        }).catch(() => { //catch error get camera
+            this.triggerStoppedScreenSharingCallbacks(localScreenCapture);
```

Comment:
Shouldn't we rethrow any error here?
Or log the error?

Reply:
I can log error again, but in getCamera function there is a catch error log.

In this case I can throw an error but we want to continue the process to stop screen sharing.

Reply:
Then I guess we should at least log the error in the console

Reply:
I think that is not necessary, the log already send in console in function getCamera. 

And like we just get promise catch, the error can’t change of the error already send in console in function getCamera.

Reply:
Sure. Still, there are a few errors that are not logged. Here for instance: http://github.com/thecodingmachine/workadventure/blob/28d687e3380ece97ba66f9aef9003f9933cf52a3/front/src/WebRtc/MediaManager.ts#L369-L376

I understand that those errors cannot be thrown in this particular case because you cannot shut down a camera if you failed to open it first.
Still, my point is that we are making the assumption that `getCamera` will log any error. If this is true today, this might not be true in the future. So I'd definitely prefer logging the error twice today than taking the risk to swallow an error message in the future.

Reply:
You harve right, I will add console error